landlab/components/landslides/tests/test_landslide.py

```diff
@@ -1,5 +1,5 @@
 """
-Unit tests for landlab.components.landslides.landslide
+Unit tests for landlab.components.landslides.landslide_probability
```

Comment:
Recommend adding a test for each of the distribution options that are specified by groundwater__recharge_distribution.